src/scripts/main.js

```diff
@@ -1,3 +1,21 @@
 'use strict';
 
-// write code here
+const spyder = document.querySelector('.spider');
+const wall = document.querySelector('.wall');
+const style = getComputedStyle(wall);
+const spyderStyle = getComputedStyle(spyder);
+
+spyder.style.height = '70px';
+
+wall.style.height = '450px';
+wall.style.width = '450px';
+wall.style.border = '15px solid #e1e1e1';
+wall.style.margin = '30px';
+wall.style.backgroundImage = 'url(images/brick-wall.svg)';
+wall.style.backgroundColor = '#efb685';
+
+spyder.style.paddingLeft = `${parseInt(style.width)
+   / 2 - parseInt(spyderStyle.width) / 2}px`;
+
+spyder.style.paddingTop = `${parseInt(style.width)
+   / 2 - parseInt(spyderStyle.width) / 2}px`;
```

Comment:
```suggestion
spyder.style.paddingLeft = `${parseInt(style.width)
   / 2 - parseInt(spyderStyle.width) / 2}px`;

spyder.style.paddingTop = `${parseInt(style.width)
   / 2 - parseInt(spyderStyle.width) / 2}px`;
```
Павук має position: absolute. Можна використати style.top та style.left і вираховувати властивості clientWidth та clientHeight у spider та wall

Reply:
что-то с файлами не так или что?

1. файл scss не подгружает стили
2. свойство clientWidth - нет в списках